src/main/kotlin/ai/digital/integration/server/common/tasks/provision/SetUpK8sClusterTask.kt

```diff
@@ -0,0 +1,26 @@
+package ai.digital.integration.server.common.tasks.provision
+
+import ai.digital.integration.server.common.constant.PluginConstant.PLUGIN_GROUP
+import ai.digital.integration.server.common.util.TerraformUtil
+import ai.digital.integration.server.common.util.TerraformUtil.Companion.getProvider
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+open class SetUpK8sClusterTask : DefaultTask() {
+
+    companion object {
+        const val NAME = "setUpK8sCluster"
+    }
+
+    init {
+        this.group = PLUGIN_GROUP
+    }
+
+    @TaskAction
+    fun launch(){
+        project.logger.lifecycle("Setting up a kubernetes cluster on ${getProvider(project)}")
+        TerraformUtil.execute(project, listOf("init"))
```

Comment:
Why not to do 
`TerraformUtil.execute(project, listOf("init", "apply"))` ?

Reply:
Terraform does not accept commands like "terraform init apply"
<img width="510" alt="Screenshot 2021-11-12 at 13 02 04" src="https://user-images.githubusercontent.com/7847140/141464062-2e82f163-4f3b-447a-b399-e8e6bcf47089.png">